types.d.ts

```diff
@@ -198,6 +208,9 @@ interface Loggers {
 
 export const loggers: Loggers;
 
+// asError
+export function asError(action: Action<any, any>): Action<any, any>;
```

Comment:
I'm not a TS user but any chance this could work?

`export function asError<P, M>(action: Action<P, M>): Action<P, M>;`

Reply:
Yeah, I felt like I was missing something when I wrote it. I just updated it to reflect any generics passed